app/models/page.rb

```diff
@@ -35,6 +35,14 @@ class Page < ApplicationRecord
 
   default_scope { order(title: :asc) }
 
+  def get_all_categories
+    Page.all.map(&:category)
```

Comment:
We should use `:pluck(:category)` which won't load in all objects. It should also probably call `.uniq` on the array.